### ISSUE TITLE
zero Reg loss and bad results

### ISSUE THREAD

**Issue by kimosalty** [zylo117/Yet-Another-EfficientDet-Pytorch#543]
I'm training from scratch of EfficientDet-d0 on a dataset which contains 600 images to recognize insulators, which is the only one class. However, when I was both training and evaluating, the Reg loss keeps zero, and the Cls loss first declines then increases and then declines rapidly. Cannot figure out what's happening, is it for lack of data and overfitting happens? But will it bring zero Reg loss? After all, I'm using the mini-version of EfficientDet. Any help, plz! 
![image](https://user-images.githubusercontent.com/31566437/98562133-ab7e4880-22e4-11eb-84d9-530e95a4c925.png)
![image](https://user-images.githubusercontent.com/31566437/98562865-7de5cf00-22e5-11eb-9ade-aa2345675aa8.png)

**Comment by zylo117** [comment]
reg loss being so low means the anchors config don't fit your targets.
You can refer to Q3 here. https://github.com/zylo117/Yet-Another-EfficientDet-Pytorch/issues/386

**Comment by myarchivedaccount** [comment]
I realize this problem due to unsuitable anchor and IOU. try modifying this line 
positive_indices = torch.ge(IoU_max, 0.4) --> positive_indices = torch.ge(IoU_max, 0.1)
Thanks

**Comment by zylo117** [comment]
> I realize this problem due to unsuitable anchor and IOU. try modifying this line
> positive_indices = torch.ge(IoU_max, 0.4) --> positive_indices = torch.ge(IoU_max, 0.1)
> Thanks

I don't think that's a proper solution, you should instead change anchor ratio to suit your datasets. @buidung2004
